common/app/implicits/Collections.scala

```diff
@@ -1,27 +0,0 @@
-package implicits
-
-import language.higherKinds
-import scala.collection.generic.CanBuildFrom
-import collection.mutable.{HashSet => MutableHashSet}
-
-trait Collections {
-
-  // thanks https://gist.github.com/1189097
-  implicit class Seq2Distinct[T, C[T] <: Seq[T]](tees: C[T]) {
-    def distinctBy[S](hash: T => S)(implicit cbf: CanBuildFrom[C[T], T, C[T]]): C[T] = {
```

Comment:
Explanation why we can delete this class in the related commit: https://github.com/guardian/frontend/pull/25190/commits/eeb25f863bb62f8842bd22acbebf05c9dc5c4f34